db/test/helper.js

```diff
@@ -421,30 +431,43 @@ exports.testEntityTable = ({
 
       setup(async function() {
         await resetTables();
-        await Promise.all(Object.values(samples).map(
-          sample => {
-            return Entity.create(sample);
-          }));
+
+        if (!notImplemented.includes('entity-functions')) {
+          await Promise.all(Object.values(samples).map(
+            sample => {
+              return Entity.create(sample);
+            }));
+        }
       });
 
-      makeTests();
+      if (notImplemented.includes('entity-functions')) {
+        test('database setup', async function() {});
+      } else {
+        makeTests();
+      }
       customTests.call(this, true);
     });
 
     suite(`db version ${dbVersion} -> ${prevVersion} preserves data`, function() {
```

Comment:
Should this test suite even bother to run if we're skipping step 1? Same comment applies to the first suite `db version ${prevVersion}`.

Reply:
Same as above.